database/schema.sql

```diff
@@ -5,28 +5,16 @@ CREATE DATABASE yumpSF;
 USE yumpSF;
 
 CREATE TABLE restaurants (
-  id INTEGER AUTO_INCREMENT PRIMARY KEY,
-  name TEXT NOT NULL 
+  id INTEGER AUTO_INCREMENT NOT NULL PRIMARY KEY,
+  restaurant_name TEXT NOT NULL 
 );
 
 CREATE TABLE dishes (
-  id INTEGER AUTO_INCREMENT NOT NULL PRIMARY KEY,
+  dish_id INTEGER AUTO_INCREMENT NOT NULL PRIMARY KEY,
   restaurant_id INTEGER NOT NULL,
-  name TEXT NOT NULL,
-  price DECIMAL(3,2) NOT NULL,
-  description VARCHAR(200) NOT NULL,
-  reviews INTEGER NOT NULL 
-);
-
-CREATE TABLE dishes_photos (
-  id INTEGER AUTO_INCREMENT NOT NULL PRIMARY KEY,
-  photos_id INTEGER NOT NULL,
-  dishes_id INTEGER NOT NULL
+  dish_name TEXT NOT NULL,
+  price DECIMAL(5,2) NOT NULL,
+  photo_url VARCHAR(200) NOT NULL,
+  number_of_photos INTEGER NOT NULL,
```

Comment:
is the "number_of_photos" how many photos a particular dish has? If so, what is the "photo_url"? That makes it look like the table only accommodates one photo. I'm unclear what's going on there. If the dish only has one photo, there's no need for a "number_of_photos" column.

Reply:
The photo_url is the link to the S3 image the previous group had for their dishes. The number_of_photos is just a "dummy-integer" without representing "real" images on a database.